index.js

```diff
@@ -11,6 +11,11 @@ module.exports = {
     "no-eval": 2,
     "no-extend-native": 2,
     "no-extra-bind": 2,
-    "no-floating-decimal": 2
+    "no-floating-decimal": 2,
+
+    // Comma
+    "comma-dangle": ["error", "always-multiline", "only-multiline"],
+    "comma-spacing": ["error", { "before": false, "after": true }],
+    "comma-style": ["error", "last"],
```

Comment:
Those two are already handled by prettier, however we can add them explicitly for ESlint as well!

Reply:
What's the point? Wouldn't eslint-config-prettier overwrite/disable the rule anyway?

Reply:
Yeah, you're right: https://github.com/prettier/eslint-config-prettier/blob/dd8511b107e5adaaa107a2f8d83d9298018e895b/index.js#L19-L90

Reply:
The difference would be that we aggregate a collection of rules we decided on / talked about instead of taking a whole set as-is. But I'm ok with first concentrating on the rules that are not present in "eslint:recommended" or "plugin:prettier/recommended".

Reply:
Ok. I'll have a look at how those configurations overwrite each other. If the "off" config from the presets supersedes our config then all my rules here will be pointless.

Reply:
The prettier config overwrites our config settings - the point of it is to turn off settings that might conflict with prettier. I think we won't need any of those rules.

Reply:
I think I have misunderstood the reason for this repository.

But fair enough.

Reply:
What did you think the reason was?

Reply:
To create our own rule set where we can decide on some coding styles instead of just taking one as-is.

Reply:
I think the idea is more to extend the existing (ember/prettier) with rules that don't collide with them (e.g prefer-const)

Reply:
I don't see a reason to diverge from what prettier is doing, even if one might not like all of it's opinions. The advantages of using prettier with default config just outweigh the disadvantages IMO.

Reply:
Trailing commas for multiline arrays/objects would make Git diffs more succinct.

Reply:
Yeah, thats what I mean: Sometimes prettiers opinions might not be ideal, but enforcing something else with toolchains like https://github.com/prettier/prettier-eslint adds lots of complexity with little benefit IMO.